bin.src/generateCoefficients.py

```diff
@@ -51,13 +52,13 @@
     # Parse start, end and timespan values.
     if args.tStart is None:
         tStart = orbits.orbits.epoch.iloc[0]
-        print "tStart was not specified: using the first epoch in the orbits file: %f" %(tStart)
+        print("tStart was not specified: using the first epoch in the orbits file: %f" % (tStart))
     else:
         tStart = args.tStart
 
     if (args.tEnd is not None) and (args.tSpan is not None):
```

Comment:
Put this before the call to readOrbits so you don't (in cases where the orbit file is long) spend time reading in the data only to find that you don't know what to do with it.